### PR TITLE
lorem-ipsum-plugin

### PR DESCRIPTION
**Pull request by moritz-hoelting**
Adds the option to copy a specified amount of sample text just by typing "lipsum".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,7 +130,9 @@ This plugin enables you to use the native macOS search to find files and folders
 
 This plugin enables you to browse through your local file system. To start you have to enter a valid absolute filepath.
 
-![file-browser](assets/feature-file-browser.png)
```
Comment (oliverschwendener) [inline README.md:133]: Did you remove this image intentionally? If so, why?

File: src/common/config/lorem-ipsum-options.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export interface LoremIpsumOptions {
+    isEnabled: boolean;
+    prefix: string;
+}
+
+export const defaultLoremIpsumOptions: LoremIpsumOptions = {
+    isEnabled: true,
```
Comment (oliverschwendener) [inline src/common/config/lorem-ipsum-options.ts:7]: As this is a plugin for very specific users, I suggest to make it disabled by default.

File: .npmrc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+node-linker=hoisted
+public-hoist-pattern=*
+shamefully-hoist=true
```
Comment (oliverschwendener) [inline .npmrc:3]: As we're using pnpm for this project, is this needed?

**Review by oliverschwendener — CHANGES_REQUESTED** [review]
Thank you for your contribution! I left some comments.

File: .npmrc (hunk 3 above)
Comment (moritz-hoelting) [reply .npmrc:3]: It was needed for me to run "pnpm package", but I will remove it from the public repository

File: README.md (hunk 1 above)
Comment (moritz-hoelting) [reply README.md:133]: No, it was a mistake, thanks for mentioning

File: src/common/config/lorem-ipsum-options.ts (hunk 2 above)
Comment (moritz-hoelting) [reply src/common/config/lorem-ipsum-options.ts]: > Thank you for your contribution! I left some comments.

Everything you commented should be fixed now. Thanks for the fast reply

**Review by oliverschwendener — APPROVED** [review]
Nice, it works as expected.

**Pull request merged**